content/posts/lets-get-lost-in arrays.md

```diff
@@ -0,0 +1,57 @@
+---
+template: post
+title: "Let's Get Lost in\_Arrays"
+slug: get-lost-in-arrays
+draft: true
+date: 2019-06-07T17:24:05.432Z
+description: Looking into the bare bones of Ruby's Array class.
+category: Programming
+tags:
+  - Ruby
+  - Java
+---
+I thought I knew what an array is, in fact, when I began learning to code it was one of the easier data structures for me to pick up. It's basically just a list of stuff with square brackets around it, `[1,2,3,4,5]`, and my writing background taught me all about lists and they're wonderful dynamicity. You can use them for anything in literature and in Ruby too, they're a wonderful multipurpose storage system; they can contain an infinite amount of stuff (lol) and can store different types of stuff (also lol). Then I began learning Java.
```

Comment:
me all about lists and their wonderful dynamicity

Reply:
You can use them for anything in literature and in Ruby too; they're a wonderful multipurpose storage system and can contain an infinite amount of stuff (lol) and in different types (also lol)